diff --git a/PCbuild/python.props b/PCbuild/python.props
index 90290f4..1704434 100644
--- a/PCbuild/python.props
+++ b/PCbuild/python.props
@@ -98,7 +97,7 @@
     ))</Field3Value>
 
     <!-- The name of the resulting pythonXY.dll (without the extension) -->
-    <PyDllName>python$(MajorVersionNumber)$(MinorVersionNumber)$(PyDebugExt)</PyDllName>
+    <PyDllName>dice_py$(PyDebugExt)</PyDllName>
 
     <!-- The version and platform tag to include in .pyd filenames -->
     <PydTag Condition="$(Platform) == 'Win32' or $(Platform) == 'x86'">.cp$(MajorVersionNumber)$(MinorVersionNumber)-win32</PydTag>
